Fix #122. Update logic to stop web app and function app only after pu…

sdubov opened the pull request:
…blisghable package is ready

sdubov committed:
Fix #122. Update logic to stop web app and function app only after publisghable package is ready

maartenba commented on PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/webapp/config/RiderWebAppRunState.kt:
```diff
@@ -136,6 +134,12 @@ class RiderWebAppRunState(project: Project,
 
+        AzureUIRefreshCore.addListener("Refresh after publish", object : AzureUIRefreshListener() {
+            override fun run() {
+
```
I assume this is empty intentionally?

sdubov replied:
Fixed. Thx. Just forgot to remove this pease of code.

maartenba reviewed (approved):
Looks good, one question (but I guess it's intentional)

sdubov committed:
Fix Azure Explorer refresh after publishing a new Function App

- Add logic to refresh Azure Explorer tree after publishing Function App
- Fix validation logic to check subscription separately for creating new and publishing to existing app
- Fix NPE exception when there are no functions under a function app

sdubov committed:
Revert compiled spark libraries
